.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,47 @@
+name: Bug Report
+description: Found a bug in our project? Create a report to help us improve.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please provide as much detail as possible.
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+      placeholder: Describe the bug here...
+    validations:
+      required: true
+
+  - type: textarea
+    id: Behavior
+    attributes:
+      label: Behavior
+      description: Describe what you expected to happen and what actually happens.
+      placeholder: |
+        Expected: ...
+        Actual: ...
+        
+    validations:
+      required: true
+
+  - type: textarea
+    id: code-example
+    attributes:
+      label: Code Example
+      description: Please add code examples if possible, so we can reproduce your steps.
+      render: php
+      placeholder: |
+        ```php
+        // Your code example here
+        ```
+
+  - type: textarea
+    id: additional-info
+    attributes:
+      label: Additional Information
+      description: Add any other context, screenshots, or details about the problem here.
+      placeholder: Additional context or screenshots here...
```

New line at end of file

```suggestion
      placeholder: Additional context or screenshots here...

```